recipes/fABBA/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "fABBA" %}
+{% set version = "0.9.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bf0672c2d0be449627380f51a7e695ddd43bf548a292de9dcfe8562e256ff95a
+
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+    
+requirements:
+  build:
+    - python >=3.6
+    - pip
+    - numpy 
+    - cython
+
+  run:
+    - python >=3.6
+    - pandas
+    - requests
+    - numpy
+    - tqdm
+    - scikit-learn
+    - scipy ==1.7.2
+    - matplotlib-base
+    
+
+test:
+  imports:
+    - fABBA
```

```suggestion
test:
  imports:
    - fABBA
  requires:
   - pip
  commands:
   - pip check
```
Because you are compiling some code, it is also good to run some tests to see that the compiled functions execute properly.

---

Thanks for mentioning that, we actually have codecov test in our repo, which works fine, so we did not arrange in conda recipe.